Kitodo/src/main/webapp/WEB-INF/templates/includes/processes/processActionsColumn.xhtml

```diff
@@ -119,15 +119,17 @@
 
             <h:commandLink action="#{ProcessListView.exportMets(process.id)}"
                            id="exportMets"
-                           styleClass="action"
-                           title="#{msgs.exportMets}"
+                           styleClass="action #{ProcessForm.hasImages(process.id) ? '' : 'disabled'}"
+                           disabled="#{not ProcessForm.hasImages(process.id)}"
```

Comment:
Should export be disabled for hierarchically superior processes, too, which do never have images?